src/pages/home/report/ReportDropUI.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {createPortal} from 'react-dom';
```

Comment:
Pretty sure we can't use `react-dom` directly in a React Native application? I know that we aren't that likely to see this come up because the drag-and-drop UI doesn't really exist in a traditional iOS or Android setting, but I think this code sets a bit of a dangerous precedent. I think there's a way to make portals work with both React.JS _and_ React Native, so we should implement that and use it.

Reply:
https://github.com/gorhom/react-native-portal looks like a viable cross-platform option that works with react-native-web

Reply:
@roryabraham  Updated to use react-native-portal, please recheck